### DISCUSSION TITLE
How to test the performance of time convergency?

### DISCUSSION THREAD

**Discussion by liuji1993** [DualSPHysics/DualSPHysics#286] [DUALSPHYSICS_V5.2]
Hi all,

I plan to test the performance of time convergence for a landslide-tsunami case with DualSPHysics. I found some descriptions for time step calculation in the XML Guide, but I’m still not sure how to modify the time step for the time convergence analysis.

 Could you please provide some suggestions? Many thanks!

![微信截图_20241223120317](https://github.com/user-attachments/assets/0e8250a5-a906-42de-969b-182a3ba27b72)

Kind regards,
Lucy

**Comment by alexbexe** [comment]
Our time step is variable, so it is computed at each time step considering maximum forces between particles, speed of sound, viscous forces, CFL, etc

Regards
Alex

**Comment by gfourtakas** [comment]
i.e. you do not need to do a "convergence" time independant study :)
